src/Components/test/E2ETest/package.json

```diff
@@ -10,7 +10,7 @@
     "sauce": "ts-node ./scripts/sauce.ts"
   },
   "author": "",
-  "license": "Apache-2.0",
```

Comment:
There are a bunch of `package.json` files with author of empty or Microsoft, should those be updated?